run

```diff
@@ -68,6 +68,38 @@ if [ ! -z "$OPENDKIM_DOMAINS" ] ; then
   dkimConfig
   service opendkim start
 fi
+
+# Checking for user specified passwd file
+if [ ! -z "$SASL_Passwds" ] ; then
+  # Creating auth settings file
+  if [ ! -e /etc/postfix/sasl/smtpd.conf ] ; then
+    cat <<'EOF' > /etc/postfix/sasl/smtpd.conf
+pwcheck_method: saslauthd
+mech_list: CRAM-MD5 DIGEST-MD5 LOGIN PLAIN
+EOF
+  fi
+  
+  # Updating saslauthd config to work with chroot
+  sed -i 's|START=no|START=yes|' /etc/default/saslauthd
+  sed -i 's|OPTIONS="-c -m /var/run/saslauthd"|OPTIONS="-c -r -m /var/spool/postfix/var/run/saslauthd"|' /etc/default/saslauthd
```

Comment:
Any gut feeling if this could break with a future base image?

Reply:
Good point. Reworked saslauthd  to run by command instead of service so that the service config doesn't need to be updated by sed. That should be a little bit safer.